tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java

```diff
@@ -815,7 +787,8 @@ public static PortalBundle getPortalBundle( IProject project ) throws CoreExcept
             }
         }
 
-        return null;
+        return new EmptyPortalBundle(
+            "Unable to get Liferay Bundle by properties 'app.server.type=" + appServerType + "'" );
```

Comment:
`Unable to determine Liferay Bundle by reading build properties 'app.serve.rtype='`